creator-node/scripts/run-tests.sh

```diff
@@ -45,3 +65,10 @@ rm -r $storagePath
 # remove test generated segments folder and .m3u8 file
 rm -rf "./test/segments"
 rm -rf "./test/testTrack.m3u8"
+
+if [ "$2" == "teardown" ]; then
```

Comment:
is this used right now? I think we can combine the standalone CN standup logic like:

```
standup_standalone_creator_node = 'false'
if (creator node is not already stood up):
     // your if [restart] block logic 
     standup_standalone_creator_node = 'true'

// set up db and execute tests

if (standup_standalone_creator_node == 'true'):
     // this teardown logic here
```

use ^ pseudo code structure and apply to bash :)

Reply:
kept the separate flag for teardown and exposed single argument to invoke per offline discussion